crates/virtio-queue/src/mock.rs

```diff
@@ -379,9 +379,12 @@ impl<'a, M: GuestMemory> MockSplitQueue<'a, M> {
         q
     }
 
-    /// Writes a single descriptor chain to the memory object of the queue, at the beginning of the
-    /// descriptor table, and returns the associated `DescriptorChain` object.
-    pub fn build_desc_chain(&self, descs: &[Descriptor]) -> Result<DescriptorChain<&M>, Error> {
+    /// Writes a multiple descriptor chains to the memory object of the queue, at the beginning of
+    /// the descriptor table, and returns the associated `DescriptorChain` object.
```

Comment:
I think it should be `returns the first DescriptorChain available` or something like that.

Reply:
Done

Reply:
You are right, I modified it.